subworkflows/local/profiling.nf

```diff
@@ -578,6 +582,74 @@ workflow PROFILING {
 
     }
 
+    if ( params.run_sylph ) {
+        ch_input_for_sylph = ch_input_for_profiling.sylph
+                                .filter{
+                                    if (it[0].is_fasta) log.warn "[nf-core/taxprofiler] sylph currently does not accept FASTA files as input. Skipping sylph for sample ${it[0].id}."
+                                    !it[0].is_fasta
+                                }
+                                .map {
+                                        meta, reads, db_meta, db ->
+                                        def db_meta_keys = db_meta.keySet()
+                                        def db_meta_new = db_meta.subMap(db_meta_keys)
+
+                                        // Split the string, the arguments before semicolon should be parsed into sylph profile
+                                        def parsed_params = db_meta_new['db_params'].split(";")
+                                        if ( parsed_params.size() == 2 ) {
+                                            db_meta_new['db_params'] = parsed_params[0]
+                                        } else if ( parsed_params.size() == 0 ) {
+                                            db_meta_new['db_params'] = ""
+                                        } else {
+                                            db_meta_new['db_params'] = parsed_params[0]
+                                        }
```

Comment:
In `usage.md` we need to document that you need to supply the `;` list in the third warning admonition of: https://nf-co.re/taxprofiler/1.2.3/docs/usage/#full-database-sheet for sylph too

Reply:
Updated the docs here too.